src/utils/eth_transaction.cairo

```diff
@@ -201,22 +206,25 @@ namespace EthTransaction {
         }
         finalize_keccak(keccak_ptr_start, keccak_ptr);
 
-        let gas_price_idx = tx_type + 1;
+        let signer_nonce_idx = 1;
+        let (signer_nonce) = Helpers.bytes_to_felt(
+            sub_items[signer_nonce_idx].data_len, sub_items[signer_nonce_idx].data, 0
+        );
         let (gas_price) = Helpers.bytes_to_felt(
-            sub_items[gas_price_idx].data_len, sub_items[gas_price_idx].data, 0
+            sub_items[signer_nonce_idx + 1].data_len, sub_items[signer_nonce_idx + 1].data, 0
         );
         let (gas_limit) = Helpers.bytes_to_felt(
-            sub_items[gas_price_idx + 1].data_len, sub_items[gas_price_idx + 1].data, 0
+            sub_items[signer_nonce_idx +2].data_len, sub_items[signer_nonce_idx +2].data, 0
```

Comment:
plz enable the github Actions in your forked repo to enable format worklow (and run `make format` locally)

Reply:
I'm able to run `make format` locally, but it seems that github Action is already enabled in my forked repo